torch/_inductor/decomposition.py

```diff
@@ -394,6 +395,48 @@ def bernoulli(self, *, generator=None):
     return torch.rand_like(self, dtype=torch.float32) < self
 
 
+@register_decomposition([aten.fmin, prims.fmin])
+def fmin(self, other):
+    return torch.where(torch.isnan(other) | (other > self), self, other)
+
+
+@register_decomposition([aten.fmax, prims.fmax])
+def fmax(self, other):
+    return torch.where(torch.isnan(other) | (other < self), self, other)
+
+
+@register_decomposition([aten.aminmax.default])
+def aminmax(self, *, dim=None, keepdim=False):
```

Comment:
this also could go to either `_refs` or `_decomp`?

Reply:
I'll move it over.